lib/build.gradle.kts

```diff
@@ -26,6 +61,8 @@ task<Jar>("sourcesJar") {
     archiveClassifier.set("sources")
 }
 
+group = "com.github.gustavkarlsson"
+version = "0.4.1"
```

Comment:
Why add these?

Reply:
This is required to publish a KMM project.
See https://kotlinlang.org/docs/reference/mpp-publish-lib.html#publish-a-multiplatform-library

In order words:
If you would publish the library to bintray (for example) **this** would be the group name and version.
So we can basically use whatever you like here. E.g. `se.gustavkarlsson.track:lib-libandroid:1.0.0`

Reply:
I know what they are used for in the general sense, but I don't use any traditional publishing method which would require it. I use jitpack and it already infers group, id and version.